### PR TITLE
[dns64] add support for DNS64

### PR DESCRIPTION
**Pull request by prfi**
This pull request adds support for DNS64. I've checked and it works on ubuntu and raspbian (I've enabled also NAT64 support on raspbian platform).

### CODE REVIEW

File: script/_dns64 [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    sudo sed -i '1a nameserver '$DNS64_NAMESERVER_ADDR $RESOLV_CONF
+}
+
+dns64_stop()
+{
+    with NAT64 || return 0
+
+    if have systemctl; then
+	    sudo systemctl stop bind9 || true
```
Comment (bukepo) [inline script/_dns64]: nit: align?

File: script/_nat64 [hunk 2]
```diff
@@ -32,20 +32,23 @@
...
-without NAT64 || test $PLATFORM = ubuntu || die "nat64 is not tested under $PLATFORM."
+without NAT64 || test $PLATFORM = ubuntu || test $PLATFORM = raspbian || die "nat64 is not tested under $PLATFORM."
 
 nat64_install()
 {
     with NAT64 || return 0
 
     test -f $TAYGA_DEFAULT -a -f $TAYGA_CONF || die 'Cannot find tayga configuration file!'
     sudo sed -i 's/^RUN="no"/RUN="yes"/' $TAYGA_DEFAULT
+    sudo sed -i 's/^IPV4_TUN_ADDR=""/IPV4_TUN_ADDR="'$TAYGA_IPV4_ADDR'"/' $TAYGA_DEFAULT
```
Comment (bukepo) [inline script/_nat64]: Are this change necessary because the configuration file already set these values?

File: script/_dns64 [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+#   Description:
+#       This script manipulates nat64 configuration.
```
Comment (bukepo) [inline script/_dns64]: `nat64` -> `dns64`

File: script/_dns64 [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    with NAT64 || return 0
+
+    test -f $BIND_CONF_OPTIONS || die 'Cannot find bind9 configuration file!'
+    sudo sed -i '/^};/i\\tallow-recursion { any; };' $BIND_CONF_OPTIONS
+    sudo sed -i '/^};/i\\t'"$DNS64_CONF" $BIND_CONF_OPTIONS
+}
+
+dns64_uninstall()
+{
+    with NAT64 || return 0
```
Comment (bukepo) [inline script/_dns64]: Involve another flag like `DNS64` to make DNS64 optional even NAT64 is enabled?
possibly `with NAT64 && with DNS64 || return 0`

File: script/_nat64 (hunk 2 above)
Comment (prfi) [reply script/_nat64]: Yes, this is necessary, without those two address are not assigned to nat64 interface, then DNS64 doesn't work.
